### PR TITLE
feat(core/search): add julian search

### PR DESCRIPTION
**Pull request by bjulian5**
Complete Slapshot: Answers SDK Core & Unit Tests codelab by adding a
"julian" search to searchAPI along with associated unit tests.

### CODE REVIEW

**Review by creotutar — APPROVED** [review]
💯 💯 💯  LGTM

File: src/core/search/searchapi.js [hunk 1]
```diff
@@ -129,6 +129,42 @@ export default class SearchApi {
...
+    const query = `${name} ${additionalSearchText}`;
+
+    let request = new ApiRequest({
+      endpoint: '/v2/accounts/me/answers/query',
+      apiKey: this._apiKey,
+      version: this._version,
+      params: {
+        'input': query,
+        'experienceKey': this._experienceKey,
+        params: {
```
Comment (creotutar) [inline src/core/search/searchapi.js]: ah sorry just caught this-- any reason this is duplicated?

Comment (bjulian5) [reply src/core/search/searchapi.js]: must have been my autocomplete, I can fix

**Review by creotutar — APPROVED** [review]
⏫ ⏫ ⏫ LGTM

**Pull request merged**